<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1175</width>
    <height>804</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Пахомов ДВ 224-321</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>491</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Параметры ГА</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Размер популяции</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="Size_population_spinBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Кол-во поколений</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="Count_General_spinBox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>20</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Вероятность мутации</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="Mutantion_doubleSpinBox">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>40</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.500000000000000</double>
     </property>
    </widget>
    <widget class="QPushButton" name="Train_GA_pushButton">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>80</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Обучение ГА</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Create_GA_pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Сгенерировать ГА</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Train_Step_pushButton">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>80</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Шаг ГА</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>501</width>
      <height>531</height>
     </rect>
    </property>
    <property name="title">
     <string>Параметры графа</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Кол-во точек</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="Count_points_spinBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
    </widget>
    <widget class="QPushButton" name="Generate_graph_pushButton">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>10</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Сгенерировать топологию</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Save_Graph_pushButton">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>40</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Сохранить</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="Start_point_spinBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Начало</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="Finish_points_spinBox">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>50</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Конец</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="Adjacency_TabletableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>471</width>
       <height>441</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="Back_Graph_pushButton">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>40</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Отмена</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="Graph_radioButton">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>10</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Граф</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="Ind_radioButton">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>30</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Индивид </string>
     </property>
    </widget>
    <widget class="QRadioButton" name="Generation_radioButton">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>50</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Поколение</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>641</width>
      <height>641</height>
     </rect>
    </property>
    <property name="title">
     <string>Отображение</string>
    </property>
    <widget class="QLabel" name="Image_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>621</width>
       <height>581</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="Graph_view_radioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Граф</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="Statistics_view_radioButton">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Статистика</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="Log_textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>660</y>
      <width>1151</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
